azurerm/helpers/azure/hdinsight.go

```diff
@@ -131,9 +131,10 @@ func FlattenHDInsightsConfigurations(input map[string]*string) []interface{} {
 		username = *v
 	}
 
+	// password is returned as `****` so let's look it up
 	password := ""
-	if v, exists := input["restAuthCredential.password"]; exists && v != nil {
-		password = *v
+	if v, ok := d.GetOk("gateway.0.password"); ok {
+		password = v.(string)
 	}
```

Comment:
since this won't be available at import time (and thus'll be empty, which won't be captured during a test) does this require a new resource after importing/a state migration?